cmd/harmony/main.go

```diff
@@ -574,6 +575,68 @@ func setupBlacklist() (map[ethCommon.Address]struct{}, error) {
 	return addrMap, nil
 }
 
+func setupViperConfig() {
+
+	// read from environment
+	envViper := viperconfig.CreateEnvViper()
+
+	//read from config file
+	configFileViper := viperconfig.CreateConfFileViper("config", "json")
```

Comment:
is this the config filename?  Can we create the config file in .hmy directory? 
like .hmy/nodeconfig.json

Reply:
yes, it's the config filename. The new commit can set the config directory and it's in ./hmy/nodeconfig.json